templates/help/starterPrograms.html.twig

```diff
@@ -1,41 +0,0 @@
-{% extends 'Default/base.html.twig' %}
```

Comment:
It seems like starter projects are not shown on the SHARE platform anymore.
Can they be completely removed? (E.g. from the admin interface)?
Or is there still a connection to the wiki somehow hidden?

@bernadettespieler @HCrane